internal/dashboard/workspace_handler.go

```diff
@@ -202,12 +203,16 @@ func (s *Server) PatchWorkspace(ctx context.Context, userId string, workspaceNam
 }
 
 func convertWorkspaceTodashv1alpha1Workspace(ws wsv1alpha1.Workspace) *dashv1alpha1.Workspace {
+	replicas := ws.Spec.Replicas
+	if replicas == nil {
+		replicas = pointer.Int64(0)
```

Comment:
Replicas is defaulted as 1 in webhook. In the case of nil, controller mamager might not be working well.
https://github.com/cosmo-workspace/cosmo/blob/main/internal/webhooks/workspace_webhook.go#L68-L72